be/src/runtime/mem_pool.cpp

```diff
@@ -45,9 +63,8 @@ MemPool::~MemPool() {
     int64_t total_bytes_released = 0;
     for (auto& chunk : chunks_) {
         total_bytes_released += chunk.chunk.size;
-        ChunkAllocator::instance()->free(chunk.chunk);
+        ChunkAllocator::instance()->free(chunk.chunk, _mem_tracker);
     }
-    mem_tracker_->Release(total_bytes_released);
```

Comment:
The memory statistics in MemPool sink into ChunkAllocator,
After the refactoring, the mem tracker is added to ChunkAllocator. In order to avoid affecting the readability of the code in two places, the statistics are unified in ChunkAllocator.